### ISSUE TITLE
Adds parameters for Google provider

### ISSUE THREAD

**Issue by loganmzz** [ronzeidman/ng2-ui-auth#78]
Authentification endpoint provided by Google accepts a lots of additionnal parameters as specified in documentation:
https://developers.google.com/identity/protocols/OpenIDConnect#authenticationuriparameters

Current optionnals:
* display
* state

Missing ones:
* prompt
* display
* login_hint
* access_type
* include_granted_scopes
* openid.realm
* hd

**Comment by ronzeidman** [comment]
Thanks,
You can add the parameters to your auth config:
```ts
export class MyAuthConfig extends CustomConfig {
    providers = {
         google: {
            optionalUrlParams: ['display', 'state', 'prompt', 'login_hint', 'access_type', 'include_granted_scopes', 'openid.realm', 'hd'],
            clientId: GOOGLE_CLIENT_ID
         }
    };
}
```
and it should work.
Please tell if this works for you and I will them by default for the next version.

**Comment by loganmzz** [comment]
It does the job ;)

**Comment by ronzeidman** [comment]
I'll keep the issue open until I add this natively in the next version.

**Comment by ronzeidman** [comment]
Included in 6.1.2
